### DISCUSSION TITLE
Pasting a beatmap url into the search bar on direct should link directly to the beatmap.

### DISCUSSION THREAD

**Discussion by nathanielcwm** [ppy/osu#16076] [Ideas] [answered]
Currently it appears like the search doesn't have any checks for beatmap urls:
![image](https://user-images.githubusercontent.com/22572406/145925002-aa23a28d-686a-4ba5-a75b-702f8e469a0b.png)
![image](https://user-images.githubusercontent.com/22572406/145925028-91e551f0-1845-45ea-8e68-2919c563a53c.png)

Having the search results show the beatmap directly or have it as the only result would be a better user experience in my opinion.

**Comment by peppy** [comment, marked as answer]
It works if you put the ID, rather than URL. Doesn't make much sense to me to paste a URL like this. Can you explain what the scenario is where you'd want to do this?

**Reply by nathanielcwm** [reply]
Can confirm that posting the ID works, well mostly just laziness since I just copied the URL in its entirety to test a specific beatmap and didn't think about copying the ID specifically.

**Reply by peppy** [reply]
Where are you copying the URL from?

**Reply by nathanielcwm** [reply]
the Z-Index issue, as I wanted to test if that map also regressed and didn't have it on my stable install.

**Reply by peppy** [reply]
Eventually we'll support dragging links into the game window, which should help with this kind of case.
